paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -143,17 +143,22 @@ private fun SemanticsNode.findAllUnmergedNodes(): List<SemanticsNode> {
   }
 }
 
-private fun SemanticsNode.accessibilityText() =
-  (
+private fun SemanticsNode.accessibilityText(): String? {
+  val stateDescription = config.getOrNull(SemanticsProperties.StateDescription)
+  val mainAccessibilityText =
     config.getOrNull(SemanticsProperties.ContentDescription)?.joinToString(", ")
       ?: config.getOrNull(SemanticsProperties.Text)?.joinToString(", ")
-      ?: config.getOrNull(SemanticsProperties.StateDescription)
-      ?: config.getOrNull(SemanticsActions.OnClick)?.label
-      ?: config.getOrNull(SemanticsProperties.Role)?.toString()
-    ).let {
+  val role = config.getOrNull(SemanticsProperties.Role)?.toString()
+  val disabled = if (config.getOrNull(SemanticsProperties.Disabled) != null) "Disabled" else null
+  val onClickLabel =
+    if (disabled != null) null else config.getOrNull(SemanticsActions.OnClick)?.label?.let { "Double tap to $it" }
```

Comment:
nit, this assumes all accessibility is in US English. We might want to think how we want to support this for other languages/Locales if we are providing our own text.

Reply:
Good call out. I think removing any provided text here for now might be the safer option, however I think adding this text in with translations in the future would be a good improvement. The on click label content on it's own often relies on the rest of the screen reader description to provide relevant context.

Reply:
Meet halfway with something like `<double-tap>: $it`?

Reply:
I updated this to `<on-click>: $it`. Since that is the field name in the `SemanticsActions` maybe this is reasonable even without translations?

Reply:
Does the same apply for Disabled? Are there any other related strings?

Reply:
Good catch! `Disabled` is the only other supplied text here, just updated it to be `<disabled>` to match the `<on-click>` styling.